R/data.R

```diff
@@ -14,7 +14,9 @@
 #' @references Locke, Adam E., et al. Genetic studies of body mass index yield new insights for obesity biology. Nature 518.7538 (2015): 197-206.
 #' @references Xue, Angli, et al. Genome-wide association analyses identify 143 risk variants and putative regulatory mechanisms for type 2 diabetes. Nature communications 9.1 (2018): 1-14.
 #' @references Sudlow, Cathie, et al. UK biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. Plos med 12.3 (2015): e1001779.
-"MI"
```

Comment:
I explicitly declared the name of the data object and then added aliases to make sure R was able to recognize the objects as documented in this manual.

Reply:
Thanks George!